js/repeater-list.js

```diff
@@ -823,57 +791,34 @@
 		}
 	}
 
-	var areDifferentColumns = function areDifferentColumns (oldCols, newCols) {
-		if (!newCols) {
-			return false;
-		}
-		if (!oldCols || (newCols.length !== oldCols.length)) {
-			return true;
-		}
-		for (var i = 0; i < newCols.length; i++) {
-			if (!oldCols[i]) {
-				return true;
-			} else {
-				for (var j in newCols[i]) {
-					if (oldCols[i][j] !== newCols[i][j]) {
-						return true;
-					}
-
-				}
-			}
-
-		}
-		return false;
-	};
-
-	var renderThead = function renderThead ($table, data) {
+	function renderThead ($table, data) {
 		var columns = data.columns || [];
 		var $thead = $table.find('thead');
+		var i, j, l, $tr;
```

Comment:
Pick better names for these index variables, or just use `.each`.  the letters `i` and `j` look too similar making this code hard to read and easy to make errors when changing it.